index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="stylesheet" href="css/font-awesome.min.css" />
+  </head>
+
+  <body>
+    <div class="nav-bar">
+      <h2>TEXT</h2>
+      <nav id="navbar">
+        <ul>
+          <li><a href="#welcome-section"> Home</a></li>
+          <li><a href="#projects"> Projects</a></li>
+          <li><a href="#contacts"> Contact me</a></li>
+        </ul>
+      </nav>
+    </div>
+
+    <div id="welcome-section">
+      <h1>TEXT</h1>
+      <p>Software developer</p>
+    </div>
+
+    <div id="projects">
+      <div class="projects-container">
+        <div class="project-tile">
+          <h3>Project 1</h3>
+          <a href="#">Link</a>
+        </div>
+        <div class="project-tile">
+          <h3>Project 2</h3>
+          <a href="#">Link</a>
+        </div>
+        <div class="project-tile">
+          <h3>Project 3</h3>
+          <a href="#">Link</a>
+        </div>
+        <div class="project-tile">
+          <h3>Project 4</h3>
+          <a href="#">Link</a>
+        </div>
+        <div class="project-tile">
+          <h3>Project 5</h3>
+          <a href="#">Link</a>
+        </div>
+        <div class="project-tile">
+          <h3>Project 6</h3>
+          <a href="#">Link</a>
+        </div>
+      </div>
+    </div>
```

Comment:
You can improve the semantic value of this piece of code by changing `div` tags for `section` tags, remember that is a best practice to use semantic tags over divs.

Reply:
Maybe that's only me, but I don't know what "semantic value" means here. Kindly rephrase the sentence to contain only understandable terms. It is worth pointing out that not all `div` tags should be replaced with `section` tags here. A phrase like "that is a best practice" is a too vague reason. Please be more specific and provide an (important and understandable) reason why the change is requested.